src/main/java/lotto/model/LottoNumber.java

```diff
@@ -1,26 +1,48 @@
 package lotto.model;
 
+import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class LottoNumber {
 
-    private List<Integer> numberlist;
+    private List<Integer> numbers;
+    private String issueType;
 
-    public LottoNumber(List<Integer> numberlist) {
-        this.numberlist = numberlist;
+    public LottoNumber(List<Integer> numbers, String issueType) {
+        validate(numbers);
+        this.numbers = numbers;
+        this.issueType = issueType;
+    }
+
+    public String getIssueType() {
+        return this.issueType;
     }
 
     public RewardStatus getRewardStatus(String[] winningNumber, int bonus) {
         int matchingCount = 0;
         for (int i = 0; i < winningNumber.length; i++) {
-            matchingCount += (numberlist.contains(Integer.parseInt(winningNumber[i]))) ? 1 : 0;
+            matchingCount += (numbers.contains(Integer.parseInt(winningNumber[i]))) ? 1 : 0;
         }
-        return new RewardStatus(matchingCount, numberlist.contains(bonus));
+        return new RewardStatus(matchingCount, numbers.contains(bonus));
+    }
+
+    public List<Integer> getLottoNumber() {
+        return numbers;
     }
 
-    public String getLottoNumbertoString() {
-        return numberlist.stream().map(String::valueOf)
-                .collect(Collectors.joining(","));
+    private void validate(List<Integer> numbers) {
+        int numberSize = (int) numbers.stream().distinct().count();
+        if (LottoNumberGenerator.LOTTO_CREATE_COUNT != numberSize
+                || LottoNumberGenerator.LOTTO_CREATE_COUNT != numbers.size()) {
```

Comment:
중복되지 않은 사이즈만 확인하면 될 것 같은데, 어떻게 생각하시나요??
```suggestion
        if (LottoNumberGenerator.LOTTO_CREATE_COUNT != numberSize) {
```

Reply:
1,2,3,4,5,5,6 의 경우 6을 반환해서 추가했습니다!!